docs/.gitignore

```diff
@@ -1,2 +1,4 @@
 .docusaurus
 build
+node_modules*
+package-lock.json
```

Comment:
Yeah, it does make sense to add these here since some contributors might not have it in a global .gitignore. 👍